README.md

```diff
@@ -67,6 +67,31 @@ sentry_dsn = "https://057006d7dfe5fff0fbed461cfca5f757@sentry.io/1111111"
 sentry_traces_sample_rate = 0.2  # 20% of requests will be logged under the performance tab
 ```
 
+`traces_sampler` can be used instead of `sentry_traces_sample_rate` to have a more granular control, [see details here](https://docs.sentry.io/platforms/rust/configuration/sampling/#configuring-the-transaction-sample-rate).
+```rust
+use rocket_sentry::RocketSentry;
+
+#[launch]
+fn rocket() -> _ {
+    let rocket_instance = rocket::build();
+    let default_rate = rocket_instance.figment().extract_inner::<f32>("sentry_traces_sample_rate").unwrap();
+    let traces_sampler = move |ctx: &TransactionContext| -> f32 {
+        if matches!(ctx.name(), "GET /specific/path/1" | "GET /specific/path/2") {
+            log::debug!("Dropping performance transaction");
+            0.
+        } else {
+            log::debug!("Sending performance transaction using the rate set in the config");
+            default_rate
+        }
+    };
+    let rocket_sentry = RocketSentry::default().set_traces_sampler(Arc::new(traces_sampler));
```

Comment:
Not sure about this mutable API for configuring the fairing...

Did you model this after some existing fairing example?

As alternative, we could consider creating a `RocketSentryOptions` struct similar to Sentry `ClientOptions`, I quite like that pattern. Or possibly also builder pattern, though not a fan of the boilerplate involved wit htat.

Reply:
I did take a look at https://api.rocket.rs/v0.5/src/rocket/shield/shield.rs#100-189. They use the enable/disable methods to configure the fairing, although its more generic than this java-esque setter ^^.  
I don't think we should go for a `RocketSentryOptions` since it'll have a single field right now and it means adding new fields will break compatibility (unless I am missing something). I'll give the builder pattern a try.

Reply:
> unless I am missing something

Ok I see that Sentry's ClientOptions have a default + with [#[non_exhaustive]](https://doc.rust-lang.org/cargo/reference/semver.html#struct-add-public-field-when-no-private) there is no compatibility break. Well, the builder is only 30 lines of boilerplate code and it makes a clean API so I would stick with it.